mobile-app/app/screens/AppNavigator/screens/Balances/BalancesNavigator.tsx

```diff
@@ -341,6 +345,59 @@ export function BalancesNavigator (): JSX.Element {
           headerBackTitleVisible: false
         }}
       />
+
+      <BalanceStack.Screen
```

Comment:
Dont we have any different approach? instead of adding new routes for Add, confirm Liquidity Screen into balance navigator?

Reply:
This was the [first approach ](https://github.com/DeFiCh/wallet/pull/2732/commits/670331166bd060c6c95420abccad1c3ac309da59), but @kyleleow and I felt that this is a better approach